public/js/interface.js

```diff
@@ -0,0 +1,60 @@
+$(document).ready(function() {
+	var game = new Game();
+	var pin = new Pin();
+	$('#frame1').text(game.rollsArray[0,1]);
+	$('#frame2').text(game.rollsArray[2,3]);
```

Comment:
Expected ']' to match '[' from line 5 and instead saw ','.<br>Expected ')' and instead saw '3'.<br>Expected an identifier and instead saw ']'.<br>Expected an assignment or function call and instead saw an expression.<br>Expected an identifier and instead saw ')'.